packages/relay-kit/src/packs/safe-4337/Safe4337Pack.ts

```diff
@@ -87,6 +87,7 @@ export class Safe4337Pack extends RelayKitBasePack<{
 
   #onchainIdentifier: string = ''
 
+  #nodeUrl?: string
```

Comment:
This rpcUrl can be obtained in several ways instead adding an additional parameter. It can be the `provider`in the initialization method or it can be extracted using the `SafeProvider` in the `protocol-kit` by retrieving the viem instance:

`protocolKit.getSafeProvider().getExternalProvider().chain?.rpcUrls[0]`